frontend/src/utils/schemas/v1.ts

```diff
@@ -12,6 +12,7 @@ export const schema_v1: JSONSchemaType<IIPFSJobMetaData> = {
     version: { type: 'integer', minimum: 1 },
     title: { type: 'string', minLength: 3, maxLength: 120 },
     description: { type: 'string', minLength: 3, maxLength: 24576 },
+    contactInformation: { type: 'string', minLength: 1, maxLength: 2457 },
```

Comment:
Should this be a maxLength of 250 to match yup?

Reply:
Yup :D